src/common/papi_web_config.py

```diff
@@ -285,14 +285,14 @@ def local_url(self) -> str:
     default_record_illegal_moves_number: int = 0
 
     # The default colors for the timers.
-    default_timer_colors: dict[int, str] = {
+    default_timer_colors: dict[int, str | None] = {
```

Comment:
Defaults are always defined, overrides are not.
The typing problem is not here, it's in the calling code.
```suggestion
    default_timer_colors: dict[int, str] = {
```